### ISSUE TITLE
Login with Github auth - internal server error

### ISSUE THREAD

**Issue by doxometrist** [LessWrong2/Lesswrong2#4038]
When trying to login with Github auth, the page processes the request for quite many seconds, then responds with 
HTTP ERROR 504

In my personal Github settings LW2.0 is visible, with the following permission
Access user email addresses (read-only)

I created LW account last week, and at first I could log in successfully

How can I login?

**Comment by nuchi** [comment]
Also unable to log in with github. In my case it's a 500 error, the text returned is merely `Internal Server Error`.

**Comment by Discordius** [comment]
Alas, weird. Works fine for me. Always annoying to debug oAuth issues.

Is there any chance you don't have an email associated with your Github account? I have a suspicion this might break login.

**Comment by nuchi** [comment]
I do in fact have an email associated

**Comment by Discordius** [comment]
Thanks! That helps me debug this.

**Comment by doxometrist** [comment]
@Discordius how is it going?

maybe I should delete my account instead?

**Comment by Discordius** [comment]
Yeah, sorry, haven't gotten around to this. @jimrandomh: Do you have the time to look into this a bit? Maybe we can find something in Sentry that helps pin it down.

**Comment by doxometrist** [comment]
or perhaps I should revoke the authorization here and restart the account somehow?
![image](https://user-images.githubusercontent.com/68142596/137076630-e4fcdb41-2447-40d5-8aa9-dddc129d5d17.png)
